app/views/AddingTripPointView.tsx

```diff
@@ -177,7 +178,8 @@ const AddingTripPointView = () => {
         );
         setTripPointCategory(
           getCategoryByName(
-            destinationDetails.superCategory?.name ?? DEFAULT_CATEGORY_NAME,
+            destinationDetails.superCategory?.name ??
```

Comment:
Is this needed? From what i understand the function already checks for the superCategory name

Reply:
It returns category with id - in request we need the whole category object. Also sometimes it happens that superCategory is null and we have to search for category in category list.